content/blog/monthly-may-2020.md

```diff
@@ -0,0 +1,92 @@
+---
+title: This Month in TiKV - May 2020
+date: 2020-05-28
+author: TiKV Authors
+---
+
+As the summer tiptoed in, we were cheerful about everything it brings to us: sunshine, ice cream, and a busy but fulfilling May!
+
+Let’s take a look at what we have accomplished.
+
+## News
+
+The first CNCF ambassador spotlight goes to Queeny Jin, who has been spreading the word about TiKV to English speakers since its inception in 2016. We appreciate Queeny’s contribution and look forward to further cooperation with her. The full blog is available [here](https://www.cncf.io/blog/2020/05/01/cncf-ambassador-spotlight-queeny-jin-of-tikv/).
```

Comment:
first?

Reply:
Yes, according to the blog, the first spotlight goes to Queeny.

Reply:
Also "further cooperation" is too serious~  We can just quote "Together we can go further"